### PR TITLE
Create Deploying-ALZ-ZTNetwork.md

### PR DESCRIPTION
**Pull request by JefferyMitchell**
## Overview/Summary

Replace this with a brief description of what this Pull Request fixes, changes, etc.

## This PR fixes/adds/changes/removes

1. *Replace me*
2. *Replace me*
3. *Replace me*

### Breaking Changes

1. *Replace me*
2. *Replace me*

## Testing Evidence

Please provide any testing evidence to show that your Pull Request works/fixes as described and planned (include screenshots, if appropriate).

### Testing URLs

The below URLs can be updated where the placeholders are, look for `{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}` & `{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}`, to allow you to test your portal deployment experience.

> Please also replace the curly brackets on the placeholders `{}`

#### Azure Public

[Deploy To Azure](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2F{YOUR GITHUB ORG/ACCOUNT HERE - Remove Curly Brackets Also}%2FEnterprise-Scale%2F{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2F{YOUR GITHUB ORG/ACCOUNT HERE - Remove Curly Brackets Also}%2FEnterprise-Scale%2F{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}%2FeslzArm%2Feslz-portal.json)

#### Azure US Gov (Fairfax)
[Deploy To Azure](https://portal.azure.us/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2F{YOUR GITHUB ORG/ACCOUNT HERE - Remove Curly Brackets Also}%2FEnterprise-Scale%2F{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2F{YOUR GITHUB ORG/ACCOUNT HERE - Remove Curly Brackets Also}%2FEnterprise-Scale%2F{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}%2FeslzArm%2Ffairfaxeslz-portal.json)

## As part of this Pull Request I have

- [ ] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [ ] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [ ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [ ] Performed testing and provided evidence.
- [ ] Ensured [contribution guidance](https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Contribution-Guide) is followed.
- [ ] Updated relevant and associated documentation.
- [ ] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

File: docs/wiki/Deploying-ALZ-ZTNetwork.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Deploy networking topology and Connectivity
+
+On the "Network Topology and Connectivity" section of the Azure landing zone portal accelerator select "Hub and spoke with Azure Firewall" radio button. Next, Select the platform connectivity subscription from the drop down. Confirm or update the address space and first networking hub region, in this case East US.
+
+![image](https://user-images.githubusercontent.com/8091766/228360733-9713f5ff-dd53-4995-b309-220442f978b5.png)
+
+Hub and spoke is the best option for Zero Trust Organizations should not have one single connection in and out of their cloud network. Network should be segmented into smaller islands with their own ingress and egress controls in minimize "blast radius".
```
Comment (jtracey93) [inline docs/wiki/Deploying-ALZ-ZTNetwork.md:11]: Is this going to get us in trouble with vWAN teams? 

Also the sentence does not make sense, please re-word. "Hub and spoke is the best option for Zero Trust **Organizations should** not have one single"

File: docs/wiki/Deploying-ALZ-ZTNetwork.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+On the "Network Topology and Connectivity" section of the Azure landing zone portal accelerator select "Hub and spoke with Azure Firewall" radio button. Next, Select the platform connectivity subscription from the drop down. Confirm or update the address space and first networking hub region, in this case East US.
+
+![image](https://user-images.githubusercontent.com/8091766/228360733-9713f5ff-dd53-4995-b309-220442f978b5.png)
+
+Hub and spoke is the best option for Zero Trust Organizations should not have one single connection in and out of their cloud network. Network should be segmented into smaller islands with their own ingress and egress controls in minimize "blast radius". 
+
+## Enable Threat protection
+
+In the next section you will want to leave the defaults to "Enable DDoS Network Protection" and "Deploy Azure Firewall" as these are pivotal requirements for threat protection. It's also recommended to select at least two avaibility zones for Azure Firewall.
```
Comment (jtracey93) [inline docs/wiki/Deploying-ALZ-ZTNetwork.md:15]:
```suggestion
In the next section you will want to leave the defaults to "Enable DDoS Network Protection" and "Deploy Azure Firewall" as these are pivotal requirements for threat protection. It's also recommended to select at least two availability zones for Azure Firewall, but ideally all 3, if the region has availability zone support.  
```

File: docs/wiki/Deploying-ALZ-ZTNetwork.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+![image](https://user-images.githubusercontent.com/8091766/228360733-9713f5ff-dd53-4995-b309-220442f978b5.png)
+
+Hub and spoke is the best option for Zero Trust Organizations should not have one single connection in and out of their cloud network. Network should be segmented into smaller islands with their own ingress and egress controls in minimize "blast radius". 
+
+## Enable Threat protection
+
+In the next section you will want to leave the defaults to "Enable DDoS Network Protection" and "Deploy Azure Firewall" as these are pivotal requirements for threat protection. It's also recommended to select at least two avaibility zones for Azure Firewall.  
+
+![image](https://user-images.githubusercontent.com/8091766/228363102-af09f069-c5f1-4be0-91e8-1050dc994bf9.png)
```
Comment (jtracey93) [inline docs/wiki/Deploying-ALZ-ZTNetwork.md:17]: Update image to show all 3 zones instead of 2

File: docs/wiki/Deploying-ALZ-ZTNetwork.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+These selections help to segment and enforce external and internal boundaries. 
+
+## Protect the Identity subscription
+
+On the "Identity" section, ensure the default (Yes) is selected on "Prevent inbound RDP from internet" and "Ensure subnets are associated with NSG".
+
+![image](https://user-images.githubusercontent.com/8091766/228366016-4eee4817-3885-491b-a064-7fdbaae9dc80.png)
+
+Here we are enforcing network protection on resources in the identity subscription like domain controller for example and what protocols can reach them with Network Security Groups.
```
Comment (jtracey93) [inline docs/wiki/Deploying-ALZ-ZTNetwork.md:27]: Not technically doing anything on rules in NSGs and therefore protocols or ports yet, so I would consider removing this as it is misleading. Just stating we enforce an NSG on the subnets in the identity sub is enough IMHO

**Review by jtracey93 — CHANGES_REQUESTED** [review]
Thanks @JefferyMitchell, nice work. Some comments in-line and below:

1. Missing from the navigation and sidebar menus so customers can find this new doc, please update.
2. Did we want to update the ALZ portal experience note shown below to point to this doc?
image
3. Can we make all images be stored in the repo instead of a githubusercontent CDN? Images for the wiki go in [`/docs/wiki/media`](https://github.com/Azure/Enterprise-Scale/tree/main/docs/wiki/media) and can be referenced in the same way as normal links etc.

File: docs/wiki/Deploying-ALZ-ZTNetwork.md (hunk 1 above)
Comment (brsteph) [reply docs/wiki/Deploying-ALZ-ZTNetwork.md:11]: @jtracey93 - There is some stuff going on with vWAN on the Zero Trust side.  I'll do a PR to adjust some of this to try and bring it in line.

Comment (brsteph) [reply docs/wiki/Deploying-ALZ-ZTNetwork.md:11]: Try https://github.com/Azure/Enterprise-Scale/pull/1267/files to see if you agree with these changes.

**Pull request closed without merging**